tests/e2e/specs/dashboard/screens.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Dashboard screens test.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { enableFeature, setupSiteKit } from '../../utils';
+
+describe( 'dashboard screens', () => {
+	it( 'ensures that the page title element pattern is correct for screens with private (and non-null) parent_slug values', async () => {
```

Comment:
Let's add one more (separate test) for the adblocker screen 👍

Reply:
Hmm, that works with Jest but AFAIK we don't have any tests that have multiple top-level describe blocks. We can however nest describe blocks which is used a fair bit.

So the structure that would be more consistent with others might be more like the following:

`screens > {screen}` where `>` indicates a nested describe, and `{screen}` might be the name or slug of the screen. Then each could have a single test like `it exists with the expected page title` (the current test descriptions are a little verbose :)

Reply:
Thanks @aaemnnosttv this has been resolved in a recent commit.